src/Tests/Domain/ClientTests/ClientAddressTests.cs

```diff
@@ -26,6 +26,6 @@ public void ClientAddress_CreateNew_SetsValues()
     public void ClientAddress_ToString_ReturnsFormattedString()
     {
         var clientAddress = ClientAddress.CreateNew("123 street", "apartment", "city", "state", "zip", "US");
-        clientAddress.ToString().ShouldBe("123 street\napartment\ncity, state zip\nUS");   
+        clientAddress.ToString().ShouldBe("123 street, apartment, city, state, zip, US");
```

Comment:
I would expect this to be formatted as:
```
"""
{Address1}
{Address2}
{City}, {State} {Zip}
{Country}
"""
```

With missing pieces not resulting in new lines